File: go/tester/src/tasks/string-length-task.go
package tasks

import (
	"strconv"
	"unicode/utf8"
)

// StringLengthTask provides type for calculating string length
type StringLengthTask struct{}

func (t StringLengthTask) Run(data []string) string {
	return strconv.Itoa(utf8.RuneCountInString(data[0]))
}

File: dotnet/01.combinatorics/Otus.Tester/Otus.Tester.ConsoleApp/Tasks/00.Strings/StringLengthTask.cs
/*
Дана строка символов.
Вычислить её длину.
 */

using Otus.Tester.ConsoleApp.Base;

namespace Otus.Tester.ConsoleApp.Tasks
{
    public class StringLengthTask : ITask
    {
        public string Run(string[] data)
        {
            return data[0].Length.ToString();
        }
    }
}
